static/app/views/discover/tags.spec.tsx

```diff
@@ -53,6 +72,7 @@ describe('Tags', function () {
       fields: [],
       sorts: [],
       query: 'event.type:csp',
+      ...commonQueryConditions,
```

Comment:
Maybe surprising that the `...` spread is _after_ the rest of the parameters, instead of before, but I guess there's no overlap in these, so it's fine?

Reply:
Yeah, no overlap. Fwiw, our linter would actually warn us of override declaration now that we use typescript

Reply:
Ooh, awesome!